packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -738,7 +738,7 @@ describe('cluster', () => {
     });
 
     // THEN
-    expect(() => cluster.addRotationSingleUser()).toThrow(/without secret/);
+    expect(() => cluster.addRotationSingleUser()).toThrow(/without master secret/);
```

Comment:
I'm also fine leaving the current error message as-is.

Reply:
"Also" means should I revert it back or is it up to me? (because you also suggested a different error message.)

Reply:
It's up to you 🙂.